test/test_cellstotex.py

```diff
@@ -30,6 +30,8 @@ def _evaluate(str_expression):
     expr = parse(definitions, MathicsSingleLineFeeder(str_expression))
     return expr.evaluate(evaluation)
 
+_evaluate('LoadModule["pymathics.asy"]')
```

Comment:
Hmmm... maybe it would be better to put this test in the pymathics repository instead of here. 

If done here, we would have to have to check to see if pymathics.asy  is installed.

Reply:
The idea would be to download and load the module in the test. But first I need to complete the package and make it works...

Reply:
If you do it in pymathics-asy then you don't have to download and everything is simpler. 

Is there a reason not to do it there?

Reply:
Yes, it could be there but is a little bit unrelated to that package. The idea of having this branch in mathics is not to actually merge it, but just to discover and collect several fixed required by the Mathics suit to load this package (and potentially, many others).  Once this package (`CellToTeX`) pass the tests, we could put this test in a different repository, with the corresponding comments about how to actually make it run.